airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -45,10 +46,16 @@ class HttpStream(Stream, ABC):
 
     source_defined_cursor = True  # Most HTTP streams use a source defined cursor (i.e: the user can't configure it like on a SQL table)
 
-    def __init__(self, authenticator: HttpAuthenticator = NoAuth()):
-        self._authenticator = authenticator
+    # TODO: remove legacy HttpAuthenticator authenticator references
```

Comment:
Maybe create a ticket for it? And when it should be removed?

Reply:
https://github.com/airbytehq/airbyte/issues/5755